modules/exploits/linux/local/systemtap_modprobe_options_priv_esc.rb

```diff
@@ -98,28 +99,24 @@ def upload_and_chmodx(path, data)
   end
 
   def check
+    return CheckCode::Safe("#{staprun_path} file not found") unless file? staprun_path
+
     # On some systems, staprun execution is restricted to stapusr group:
     # ---s--x---. 1 root stapusr 178488 Mar 28  2014 /usr/bin/staprun
-    unless cmd_exec("test -x '#{staprun_path}' && echo true").include? 'true'
-      vprint_error "#{staprun_path} is not executable"
-      return CheckCode::Safe
-    end
+    return CheckCode::Safe("#{staprun_path} is not executable") unless cmd_exec("test -x '#{staprun_path}' && echo true").include?('true')
```

Comment:
An `executable?` method was added for this purpose after this module was written.

https://github.com/rapid7/metasploit-framework/blob/c2e593124827c045a46bf5d8338d251e09c3cea5/lib/msf/core/post/file.rb#L234-L238

```suggestion
    return CheckCode::Safe("#{staprun_path} is not executable") unless executable?(staprun_path)
```